src/openrct2/ride/Vehicle.h

```diff
@@ -524,8 +524,8 @@ enum
 enum
 {
     SOUND_RANGE_SCREAMS_0 = 0,
```

Comment:
Given number 1 and 2 are renamed to match the array they represent, I suggest we rename this one to `SOUND_RANGE_SCREAMS_MISC` as well, so it is consistent again.

Reply:
Let’s make sure it is playtested first. Broxzier’s criticism was that the initial version did not sound right for water rides.

Reply:
Sure, no need to rush updating the replay tests before we're satisfied, indeed.